code/_onclick/cyborg.dm

```diff
@@ -72,6 +72,8 @@
 
 		W.attack_self(src)
 		return
+	if(A.type==/obj/structure/flora/pottedplant || A.parent_type==/obj/structure/flora/pottedplant)
```

Comment:
This means borgs literally cannot click on potted plants at all.

Reply:
I don't think that's an issue in and of itself, considering rn only interaction with potted plants is putting stuff into them. However the WAY it was done is very much dumb.